econml/utilities.py

```diff
@@ -1871,3 +1872,42 @@ def fit(self, X, Y):
     def predict(self, X):
         predictions = self.model.predict(X)
         return reshape(predictions, (-1, 1)) if self.needs_unravel else predictions
+
+
+class WeightedLassoCVWrapper:
+    """Helper class to wrap either WeightedLassoCV or WeightedMultiTaskLassoCV depending on the shape of the target."""
+
+    def __init__(self, *args, **kwargs):
+        self.args = args
+        self.kwargs = kwargs
+
+    def fit(self, X, y):
+        assert shape(X)[0] == shape(y)[0]
```

Comment:
these asserts are not necessary since the underlying `self.model` model does these checks internally. Also, I recommend raising exceptions with messages rather than simple asserts.